federation_child/api.py

```
@@ -338,4 +390,237 @@
+
+    fed_sett=frappe.get_doc("Federated Erp Setting")
+    url=f'{fed_sett.federated_site_name}/api/method/fedration_erp.fedration_erp.api.create_role'
+    payload=json.dumps({
+            "records":records
+        },default=str)
+    api_secret =fed_sett.get_password(fieldname="api_secret_pass", raise_exception=False)
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization':'token '+str(fed_sett.api_key)+":"+str(api_secret)
+    }
+    response = requests.request("Post", url, headers=headers,data=payload)
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)